opacclient/gradle.properties

```diff
@@ -11,9 +11,10 @@
 # The setting is particularly useful for tweaking memory settings.
 # Default value: -Xmx10248m -XX:MaxPermSize=256m
 # org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-org.gradle.jvmargs=-Xmx2048m
+#org.gradle.jvmargs=-Xmx2048m
 
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+org.gradle.jvmargs=-XX\:MaxHeapSize\=256m -Xmx256m
```

Comment:
Can you tell me why you removed the line above and changed this one? Did you run out of memory? We need to find a solution for this to work nicely on all developers computers.

We'll find some way to have this modified locally. You could try if it works when you revert these changes and insert your custom line into the ``local.properties`` file. I haven't tried that so far.

For now, it would be nice if you kept this change only to your local file. I suggest reverting this change and then using "git add --patch" to only partially add files to the commit to avoid committing it again.